spec/GraphQLOverHTTP.md

```diff
@@ -170,9 +170,14 @@ http://example.com/graphql?query=query(%24id%3A%20ID!)%7Buser(id%3A%24id)%7Bname
 
 Note: {query} and {operationName} parameters are encoded as raw strings in the query component. Therefore if the query string contained `operationName=null` then it should be interpreted as the {operationName} being the string `"null"`. If a literal `null` is desired, the parameter (e.g. {operationName}) should be omitted.
 
+GET requests MUST only be used for executing queries. If the values of {query} and {operationName} indicates that a non-query operation is to be executed, the server SHOULD immediately respond with error status code 405 Method Not Allowed and halt execution.
```

Comment:
Can you factor out this part into a separate PR? That would allow us to get the minor edits through the door faster and have a focused discussion on this substantial change.

Reply:
Sure. Other PR to follow.